homeassistant/components/media_player/__init__.py

```diff
@@ -82,6 +83,7 @@
 SUPPORT_PLAY_MEDIA = 512
 SUPPORT_VOLUME_STEP = 1024
 SUPPORT_SELECT_SOURCE = 2048
+SUPPORT_ENQUEUE = 4096
```

Comment:
No need for making it part of support, other methods will just ignore the passed in variable.